user_guide_src/source/concepts/factories.rst

```diff
@@ -334,14 +334,14 @@ Set the following property to ``true`` in **app/Config/Optimize.php**::
 
     public bool $configCacheEnabled = true;
 
-Since v4.5.0, you can enable this with the ``spark optimize`` command.
+Or you can enable this with the ``spark optimize`` command.
```

Comment:
I think it is better to keep "Since v4.5.0", because devs who use old CI4 version could know they cannot use the feature when reading the latest user guide.

Reply:
How to Enable Config Caching have info version added 4.5.0, I think that includes 1 segment
![image](https://github.com/user-attachments/assets/2dcbdb9f-33e7-4e1e-b9ed-91bc6076ea86)

Reply:
Okay.